app/controllers/rules_controller.rb

```diff
@@ -7,9 +7,12 @@ def index
   end
 
   def show
-    rule = ::Pundit.policy_scope(User.current, ::Rule).friendly.find(
-      params[:id]
-    )
+    rule = ::Pundit.policy_scope(User.current, ::Rule).where(
+      'rules.slug LIKE ?',
+      "%#{ActiveRecord::Base.sanitize_sql_like(params[:id])}%"
```

Comment:
Let's also be sure not to break our API here when requesting a Rule by ID rather than slug.